pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/InstallVirtualEnvironmentTask.groovy

```diff
@@ -84,6 +88,45 @@ class InstallVirtualEnvironmentTask extends DefaultTask implements FailureReason
             }
         })
         project.delete(packageDir)
+        buildPipConfFile()
+    }
+
+    /**
+     * Writes a new pip.conf file.  You can configure the contents of this file with the python extension
+     * or leave it blank for it to pick up the pip.conf in your system properties.
+     *
+     * index-url = https://pypi.python.org/simple/
+     * @return
+     */
+    private buildPipConfFile() {
+        def fileExtension
+
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            fileExtension = "ini"
+        } else {
+            fileExtension = "conf"
+        }
+
+        File pip = new File(Paths.get(pythonDetails.getVirtualEnv().absolutePath, "pip.${fileExtension}").toString())
+        if (!pip.exists()) {
+            println("creating pip.${fileExtension}")
+            pip.createNewFile()
+            PythonExtension pythonExtension = project.extensions.getByName("python") as PythonExtension
+
+            Map<String, Map<String, String>> pipConfig = pythonExtension.pipConfig
+
```

Comment:
the config or the whole feature?  Im still having problems getting pip to read the file for some reason too.  I'm beginning to think its the combination of pip 7.X with python 3.5 that im running is whats screwing me up.